controllers/tools.js

```diff
@@ -225,14 +229,17 @@ router.all('/', function (req, res, next) {
     const itemsPerPage = (req.query.limit || 10);
```

Comment:
Big code duplication with /search ... create a function. Changes need to be made only once